.structure-config

```diff
@@ -0,0 +1 @@
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```

Comment:
`.coveragerc` is part of anaconda-34.25.0.9-1.el9.src.rpm - is it expected? If yes, then is should not be marked as a infrastructure file.

Reply:
The same answer as in https://github.com/rhinstaller/anaconda/pull/3462#discussion_r660553140 .